tests/cli/test_cli_smoke.py

```
@@ -0,0 +1,106 @@
+_SETUP_PATCH = "file_organizer.cli.organize._check_setup_completed"
+
+
+class TestOrganizeSmoke:
+    """fo organize --dry-run exits 0 and reports processed count."""
+
+    # FileOrganizer is lazy-imported inside organize(); patching at definition
+    # site because cli.organize holds no module-level reference to the class.
+    @patch("file_organizer.core.organizer.FileOrganizer")
+    @patch(_SETUP_PATCH, return_value=True)
+    def test_organize_dry_run_exits_zero(
+        self, _setup: MagicMock, mock_cls: MagicMock, tmp_path: object
```

Ruff (0.15.10)

[warning] Fixture `_setup` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)